documentation/book/proc-renewing-ca-certs-manually.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='proc-renewing-ca-certs-manually-{context}']
+
+= Renewing CA certificates manually
+
+If automatic CA certificate generation is enabled (`generateCertificateAuthority=true`), the cluster CA and clients CA certificates will auto-renew at the start of their respective certificate renewal periods. You can manually renew one or both of these certificates before the certificate renewal period begins, if required for security reasons. A renewed certificate uses the same private key as the old certificate.
+
+.Prerequisites
+
+* The Cluster Operator is running.
+* A Kafka cluster in which CA certificates and private keys are installed.
+
+.Procedure
+
+. View a description of the certificate authority `Secret` that contains the CA certificate that you want to renew.
```

Comment:
Why is the user supposed to view the Secret? Seem unnecessary / this step doesn't explain it.